site/src/components/mdx/mdx-page-content.tsx

```diff
@@ -139,16 +140,22 @@ export const MdxPageContent: FunctionComponent<MdxPageContentProps> = ({
               ? `#${headingAtScrollPosition.anchor}`
               : ""
           }`,
+          undefined,
+          {
+            scroll: false,
```

Comment:
At this point we update the routes based on the sections users get to while scrolling, so we don't need the scroll behaviour of next/router